mappings/net/minecraft/item/OnAStickItem.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_1758 net/minecraft/item/OnAStickItem
```

Comment:
hmm, maybe like steering item, as the on a stick thing can actually act like a steering wheel

Reply:
what about ComposedStickItem, or LuredStickItem if that makes any sense?

Reply:
Think the main functionality is still steering than composed stick, otherwise `on a stick item` is fine

Reply:
I think SteeringItem is too far away from the item names